src/massive/munit/client/RichPrintClient.hx

```diff
@@ -223,11 +223,23 @@ class RichPrintClient extends PrintClientBase
 		external.trace(t);
 	}
 	
+	#if (flash && air)
+	private var flashOutput = "";
+	#end
+	
 	override public function print(value:Dynamic)
 	{
 		super.print(value);
 		#if(neko || cpp || java || cs || python || php || hl || eval)
 		Sys.print(value);
+		#elseif (flash && air)
+		flashOutput += value;
+		while (flashOutput.indexOf("\n") > -1)
```

Comment:
Why not tracing as-is?

Reply:
I believe the issue is that Flash trace supports only one line at a time